ingest/validation/validate_metadata.py

```diff
@@ -948,6 +948,7 @@ def validate_input_metadata(metadata, convention, bq_json=None):
     """Wrapper function to run validation functions
     """
     collect_jsonschema_errors(metadata, convention, bq_json)
+    print(metadata.issues)
```

Comment:
This looks like it was added for debugging? Please remove before merge.

Reply:
553dd81